src/ionosphere/waccmx/edyn_init.F90

```diff
@@ -52,7 +53,8 @@ subroutine edynamo_init(mpicomm, nlon_in, nlat_in, nlev_in,                 &
 
 #ifdef WACCMX_EDYN_ESMF
       call alloc_edyn()      ! allocate dynamo arrays
-      call edyn_esmf_init(mpicomm)       ! initialize ESMF
+!      call edyn_esmf_init(mpicomm)       ! initialize ESMF
+      call edyn_grid_comp_init(mpicomm)
```

Comment:
I guess we should remove the commented out code.

Reply:
Yeah, sorry again about not cleaning up.